src/uct/tcp/tcp_sockcm_ep.c

```diff
@@ -298,11 +438,32 @@ ucs_status_t uct_tcp_sockcm_ep_recv(uct_tcp_sockcm_ep_t *cep)
     return status;
 }
 
+ucs_status_t uct_tcp_sockcm_ep_set_sockopt(uct_tcp_sockcm_ep_t *ep)
+{
+    ucs_status_t status;
+
+    ep->sndbuf  = UCS_MEMUNITS_AUTO;
```

Comment:
what about having this in configuration?
looks like, it will be very large value in case of `AUTO=(size_t)(-2)`
I'd propose smth like we have in TCP:
https://github.com/openucx/ucx/blob/93540351a13b691fad620ad2f80abb9587ed8134/src/uct/tcp/tcp_iface.c#L292

Reply:
@dmitrygx but in tcp_iface.c it is also set to this value - its default value -

Reply:
@alinask yes, it uses the default value.
but it handles this case here - https://github.com/openucx/ucx/blob/1a4906a947590de08983ed48a07260933d005a77/src/uct/tcp/tcp_iface.c#L301
i.e. if configuration value is AUTO, it just doesn't set any value and uses system configured value